cli/azd/cmd/init.go

```diff
@@ -314,54 +316,13 @@ func (i *initAction) Run(ctx context.Context) (*actions.ActionResult, error) {
 	case initProject:
 		tracing.SetUsageAttributes(fields.InitMethod.String("project"))
 
-		composeAlphaEnabled := i.featuresManager.IsEnabled(composeFeature)
-		if !composeAlphaEnabled {
-			err = i.repoInitializer.InitializeMinimal(ctx, azdCtx)
-			if err != nil {
-				return nil, err
-			}
-
-			_, err := i.initializeEnv(ctx, azdCtx, templates.Metadata{})
-			if err != nil {
-				return nil, err
-			}
-
-			followUp = ""
-		} else {
-			fi, err := os.Stat(azdCtx.ProjectPath())
-			if err != nil && !errors.Is(err, os.ErrNotExist) {
-				return nil, err
-			}
-
-			if fi != nil {
-				return nil, fmt.Errorf("project already initialized")
-			}
-
-			name, err := i.console.Prompt(ctx, input.ConsoleOptions{
-				Message:      "What is the name of your project?",
-				DefaultValue: azdcontext.ProjectName(azdCtx.ProjectDirectory()),
-			})
-			if err != nil {
-				return nil, err
-			}
-
-			prjConfig := project.ProjectConfig{
-				Name: name,
-			}
-
-			if composeAlphaEnabled {
-				prjConfig.MetaSchemaVersion = "alpha"
-			}
-
-			err = project.Save(ctx, &prjConfig, azdCtx.ProjectPath())
-			if err != nil {
-				return nil, fmt.Errorf("saving project config: %w", err)
-			}
-
-			followUp = "Run " + output.WithHighLightFormat("azd add") + " to add new Azure components to your project."
+		err = i.repoInitializer.InitializeMinimal(ctx, azdCtx)
```

Comment:
The previous 'else' block (when composeAlphaEnabled was true) has been moved into `InitializeMinimal(ctx, azdCtx)`